addons/google_calendar/static/tests/google_calendar_tests.js

```diff
@@ -100,6 +100,8 @@ QUnit.module('Google Calendar', {
                     return Promise.resolve();
                 } else if (route === '/web/dataset/call_kw/res.partner/get_attendee_detail') {
                     return Promise.resolve([]);
+                } else if (route === '/web/dataset/call_kw/hr.employee/search') {
+                    return Promise.resolve([]);
```

Comment:
I'm guessing this is necessary for the enterprise part?
Would be nice to give a bit more context as to where it's used in the commit message.

(This probably "conflicts" with the task that removes HR dependency from calendar, we will have to be careful about that).

Reply:
Side note: I think you forgot to address this?

Reply:
Additional note: in commit message, "route that search" -> "route that searches" ^^

Would also say "correctly mock all routes in test" maybe as commit header ?